_overviews/cheatsheets/index.md

```diff
@@ -501,10 +498,9 @@ println(upper.getOrElse(""))</code></pre></td>
       <pre class="highlight"><code>option match {
   case Some(x)
     if f.isDefinedAt(x) =&gt; ...
-  case Some(_)          =&gt; None
-  case None             =&gt; None
+  case _                =&gt; None
```

Comment:
Yes, it could be minimized with the wildcard pattern match, but the extra pattern match cases helped explain `collect`, IMO.